### PR TITLE
Mark unpullable

### PR DESCRIPTION
**Pull request by AliengirlLiv**
Let admins mark rooms as unpullable

### CODE REVIEW

File: app/views/admin/map.html.erb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+<button type="button" class="btn" data-toggle="modal" data-target="#preplaced">Mark Room as Pullable/Unpullable</button>
+
+<div id="preplaced" class="modal fade" role="dialog">
+  <div class="modal-dialog">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal">&times;</button>
+      </div>
+      <div class="modal-body">
+        <%= form_tag("/admin/map", method: "edit_mark", :id => "yolo") do |f| %>
```
Comment (forLostLense) [inline app/views/admin/map.html.erb]: Remove yolo?

File: app/controllers/admin_controller.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
+class AdminController < ApplicationController
+  def map
+  end
+
+  def edit_mark
+    if params[:commit] == "Delete Mark (make pullable)"
```
Comment (forLostLense) [inline app/controllers/admin_controller.rb]: Overall LGTM, except for `params[:commit]`. If it's an input by admin, `"Delete Mark (make pullable)"` seems too long for verification.

Comment (AliengirlLiv) [reply app/controllers/admin_controller.rb]: Fixed! I now use a "delete" variable

File: app/views/admin/map.html.erb (hunk 1 above)
Comment (AliengirlLiv) [reply app/views/admin/map.html.erb]: fixed!

**Pull request closed without merging**